src/popup/router/components/Header.vue

```diff
@@ -121,8 +121,10 @@ export default {
     ...mapMutations(['setNotificationsStatus']),
     back() {
       const fallBackRoute = this.isLoggedIn ? '/account' : '/';
+      let { fullPath } = this.$route;
+      fullPath = fullPath.endsWith('/') ? fullPath.slice(0, -1) : fullPath;
```

Comment:
Isn't it a bad practice to mutate router object props?

Reply:
I'm making destructuring assignment of string, it'll just copy the value